it/v1/endpoints/RegisterNinoISpec.scala

```diff
@@ -43,16 +43,34 @@ class RegisterNinoISpec extends IntegrationBaseSpec {
 
   "Calling the /api/register endpoint" when {
 
-    "the feature switch is on" when {
+    "the useAuth feature switch is off" should {
+      "not call the auth service" in new Test {
```

Comment:
If it is called, since it is not mocked out, it will fail. Though I suppose there's a mocking function to ensure something isn't called. I'll add that in.